examples/with-storybook/package.json

```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook -s public/",
+    "serve-storybook": "serve storybook-static"
```

Comment:
How is `serve-storybook` different from `storybook`? they both server storybook and show how it works for the example, right?

Reply:
`serve-storybook`run the built version of Storybook. It demonstrate a production deployment.
`storybook` alone is a development server, it will listen for file change etc. and is not suited for production.

This is not a "minimal" example but since Storybook is meant for team work with UX team I though it could be good to demo this feature + how to handle `/public` folder.